packages/tree-view-for-tests/src/__tests__/tree.test.ts

```diff
@@ -6,33 +6,12 @@ test('it can serialise a folder', () => {
   // so can't include any .test.ts files that actually use this.
   // I picked one of our packages instead.
 
-  expect(tree(path.join(__dirname, '../../../create-modular-react-app')))
+  expect(tree(path.join(__dirname, '../../../eslint-config-modular-app')))
```

Comment:
I think you'll want to rebase this as I already updated it to point to itself in a meta-testing kind of fashion.

Reply:
I think eslint-config-modular-app is a more stable folder to point to. It doesn't have tests with snapshots and tree-view tests does.

Reply:
I just checked the last PRs and this change didn't make it in there.

Reply:
Ah - it was this PR which I updated it in: https://github.com/jpmorganchase/modular/pull/492. I think the only issue is for me that these will eventually change so then we'll get "unrelated failures" where as using the tree folder keeps it bound together. I'm happy to merge this though for now.

Reply:
I'm going to do the proper thing and just set up and tear down a test dir for this...